sdk/healthinsights/azure-healthinsights-radiologyinsights/samples/sample_age_mismatch_inference.py

```diff
@@ -0,0 +1,160 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+import datetime
+import os
+import uuid
+
+
+from azure.core.credentials import AzureKeyCredential
+from azure.healthinsights.radiologyinsights import RadiologyInsightsClient
+from azure.healthinsights.radiologyinsights import models
```

Comment:
I would move these imports into the main sample space so customers can see what they need to import from the library

Reply:
Same feedback applies to other samples